src/gluonts/transform/split.py

```diff
@@ -395,7 +347,7 @@ def flatmap_transform(
                 past_ts = past_ts.transpose() if self.output_NTC else past_ts
                 d[self._past(ts_field)] = past_ts
```

Comment:
BTW, one thing that would be nice here, is to support incomplete prediction windows. Right now we only support full time windows in the future. If we mask the loss using a `future_is_pad` indicator, we can also support incomplete future windows. This would be quite useful when you have short time series (the NBEATS paper does this). It is also more consistent with samplers being able to pick any point from the time series without constraints.

Reply:
yes, that could be an option. maybe though we address it separately? we can open a dedicated issue